automation/taskcluster/decision_task.py

```diff
@@ -83,7 +83,16 @@ def pr_or_push(is_push=False):
             for craft_function in (
                 BUILDER.craft_raptor_speedometer_task,
                 BUILDER.craft_raptor_speedometer_power_task,
-                BUILDER.craft_raptor_tp6m_task
+                BUILDER.craft_raptor_tp6m_task(for_suite=1),
```

Comment:
Thanks to your nice solution, I believe we can factorize these 10 lines into:

``` py
ALL_RAPTOR_CRAFT_FUNCTIONS = [
    BUILDER.craft_raptor_tp6m_task(for_suite=i)
    for i in range (1, 11)
] + [
    BUILDER.craft_raptor_speedometer_task,
    BUILDER.craft_raptor_speedometer_power_task
]

for craft_function in ALL_RAPTOR_CRAFT_FUNCTIONS:
```

What do you think?

Reply:
Yes, this does make the code DRYer. I will go with your suggestion.